renovate.json5

```diff
@@ -0,0 +1,24 @@
+{
+  "extends": [
+    "github>cybozu/renovate-config",
+    ":prConcurrentLimitNone"
+  ],
+  "npm": {
+    "packageRules": [
+      {
+        // automerge minor/patch updates of devDependencies
+        "matchPackagePatterns": [
+          "*"
+        ],
+        "matchDepTypes": [
+          "devDependencies",
+        ],
+        "matchUpdateTypes": [
+          "minor",
+          "patch"
```

Comment:
@mshrtsr 

Because we already used the presets ":automergePatch", so do we need to specify "patch" in this "packageRules"?

Reply:
@hung-cybo 
Exactly! I fixed it.

IMO: It is not good that patch update is implicitly declared in the inherited source, so it might be better to stop using `cybozu/renovate-config` in the future.